pocs/serial_handlers/protocol_arduinosimulator.py

```diff
@@ -30,14 +30,17 @@ class ArduinoSimulator:
     at a rate similar to 9600 baud, the rate used by our Arduino sketches.
     """
 
-    def __init__(self, message, relay_queue, json_queue, stop, logger):
+    def __init__(self, message, relay_queue, json_queue, chunk_size, stop, logger):
         """
         Args:
             message: The message to be sent (millis and report_num will be added).
-            relay_queue: The queue.Queue instance from which replay command bytes are read
-                and acted upon. Elements are of type bytes.
-            json_queue: The queue.Queue instance to which json messages (serialized to bytes)
-                are written at ~9600 baud. Elements are of type bytes.
+            relay_queue: The queue.Queue instance from which relay command
+                bytes are read and acted upon. Elements are of type bytes.
+            json_queue: The queue.Queue instance to which json messages
+                (serialized to bytes) are written at ~9600 baud. Elements
+                are of type bytes (i.e. each element is a sequence of bytes of
+                length up to chunk_size).
+            chunk_size: The number of bytes to write to json_queue at a time.
```

Comment:
No default for this?

Reply:
Yes, but not here, instead down in _create_simulator.